### ISSUE TITLE
Confuse background color on checkbox group

### ISSUE THREAD

**Issue by muhammaddicky** [stisla/stisla#110]
Background color on **checkbox group** in [forms/advanced form](https://demo.getstisla.com/forms-advanced-form.html) has same **color** between **check** and **focus** state, so when user uncheck that checkbox, user will still see same **color** after uncheck until user move cursor to focus on another place. this can make confuse some user.

https://github.com/stisla/stisla/blob/096e834c00641395e4d5ebeddda4cf020d61adf0/sources/scss/override/_form.scss#L212
```
selectgroup-input {
  &:focus + .selectgroup-button,
  &:checked + .selectgroup-button {
    background-color: color(primary);
    color: #fff;
    z-index: 1;
  }
}
```

**Comment by muhammaddicky** [comment]
#111

**Comment by nauvalazhar** [comment]
Hi, thank you for reporting this issue. Do you mean this part?
![image](https://user-images.githubusercontent.com/14899175/70773347-cb8e4a80-1da9-11ea-9c39-7fe54d639e81.png)

![image](https://user-images.githubusercontent.com/14899175/70773413-fd9fac80-1da9-11ea-8144-902533f33fe2.png)

**Comment by muhammaddicky** [comment]
@nauvalazhar yes

**Comment by muhammaddicky** [comment]
Issue fix #111
